src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -55,6 +57,16 @@
     <None Include="project.json" />
     <None Include="PackageFiles\**\*.*" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ApiCompat\ApiCompat.csproj">
```

Comment:
Just so it is clear by only adding it here we will only use it when running on .NET Core msbuild which we currently only do for x-plat. We will need to have a solution for the net45/full msbuild runs as well.

I was chatting with @mellinoe about this and we think perhaps having both the task and the exe and toggle between them as necessary. In otherwords when we can use this task library use the task but in places we cannot call the exe.